packages/esm-service-queues-app/src/queue-table/cells/queue-table-action-cell.component.tsx

```diff
@@ -24,8 +35,8 @@ export function QueueTableTransitionCell({ queueEntry }: QueueTableCellComponent
   );
 }
 
-export const queueTableTransitionColumn: QueueTableColumn = {
+export const queueTableActionColumn: QueueTableColumn = {
```

Comment:
Is this a type annotation? If so, can we annotate it using either `type` or `interface`? Otherwise, if it's a variable, can we make it title-case?